packages/form/src/form-item.vue

```diff
@@ -54,6 +54,7 @@ import { elFormEvents, elFormItemKey, elFormKey } from './token'
 import type { PropType } from 'vue'
 import type { ElFormContext, ValidateFieldCallback } from './token'
 import type { FormItemRule } from './form.type'
+import type { CSSProperties } from 'vue'
```

Comment:
Minor: you can add this member at line#54

Reply:
@qige2016 Please address this one so that we can get this merged.

Reply:
@JeremyWuuuuu Thanks for reviewing.